code/week6/CMakeLists.txt

```diff
@@ -1,11 +1,6 @@
 cmake_minimum_required(VERSION 3.5)
 project(week6)
-find_package(OpenCV REQUIRED)
 
 set(CMAKE_CXX_STANDARD 14)
 
-add_executable(main main.cpp)
-add_executable(findBlue findBlue.cpp)
-
-target_link_libraries(main ${OpenCV_LIBS})
-target_link_libraries(findBlue ${OpenCV_LIBS})
+add_executable(ttt main.cpp Board.cpp Board.h)
```

Comment:
I don't think header files should be added to the executable targets.

Reply:
Good catch. CLion auto adds them as part of the new file wizard.